baseplate/events/queue.py

```diff
@@ -21,8 +21,25 @@
 import time
 import uuid
 
+from enum import Enum
+
 from . import MAX_EVENT_SIZE, MAX_QUEUE_SIZE
 from baseplate.message_queue import MessageQueue, TimedOutError
+from baseplate._utils import warn_deprecated
+
+
+class FieldType(Enum):
+    NORMAL = None
+    """
+    For fields containing sensitive information like IP addresses that must
```

Comment:
oh, i think you'll also need to add an `autoclass` thing to docs/baseplate/events.rst to get this class into the docs